src/typescript/frontend/src/components/pages/emojicoin/components/chat/ChatBox.tsx

```diff
@@ -118,6 +120,22 @@ const ChatBox = (props: ChatProps) => {
     /* eslint-disable react-hooks/exhaustive-deps */
   }, [props.data.chats.length, chats.length]);
 
+  useEffect(() => {
+    (async () => {
+      const names = await getANSNames(sortedChats.map((chat) => chat.user));
```

Comment:
```suggestion
      const users = sortedChats.map((chat) => chat.user);
      const names = await getANSNames(users);
```

Another small readability thing, I know it's a bit more verbose but it's just a lot easier to read